src/utils/check-value.js

```diff
@@ -1 +1,3 @@
 export const isNullable = (value) => (value === null || value === undefined);
+
+export const isNullableOrEmpty = (value) => (isNullable(value) || value === '' || value === []);
```

Comment:
last statement don't have any sense. It will always return `false`

Reply:
Thanks, addressed